generator/tsconfig.json

```diff
@@ -4,9 +4,10 @@
     "moduleResolution": "node",
     "checkJs": true,
     "allowJs": true,
+    "noEmit": true,
     "strict": true,
     "resolveJsonModule": true,
-    "lib": ["es2015", "es2017.object", "esnext", "dom"]
+    "lib": ["esnext", "dom"]
```

Comment:
I was getting an error about bad targets & stuff, and this seemed to resolve it. The `es*` libs implicitly include all the previous versions.